Backend-API/src/main/java/com/avangers/backendapi/controllers/UserController.java

```diff
@@ -3,23 +3,28 @@
 
 import com.avangers.backendapi.DTOs.RegisterUserDTO;
 import com.avangers.backendapi.services.UserService;
+import com.avangers.backendapi.services.UserServiceResponse;
 import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-@CrossOrigin("http://localhost:5173/")
 @RestController
+@CrossOrigin("http://localhost:5173/")
 @RequiredArgsConstructor
 public class UserController {
 
     private final UserService userService;
 
     @PostMapping("/register")
     public ResponseEntity<String> register(@Valid @RequestBody RegisterUserDTO registerUserDTO) {
```

Comment:
I do not really like returning with anything other than JSON here. Can we create a `ReponseDTO` class/record to make things deserialisable on the frontend and other clients?